helm/stargate/templates/stargate_coordinator_deployment.yaml

```diff
@@ -0,0 +1,85 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: stargate-coordinator
+  labels:
+    app: coordinator
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app: coordinator
+  template:
+    metadata:
+      labels:
+        app: coordinator
+    spec:
+      containers:
+      - name: coordinator
+        {{ if eq .Values.cassandra.clusterVersion "3.11" }}
+        image: "{{ .Values.image.repository311 }}:{{ .Values.image.tag }}" 
+        {{ else if eq .Values.cassandra.clusterVersion "6.8" }}
+        image: "{{ .Values.image.repositoryDse68 }}:{{ .Values.image.tag }}"
+        {{ else }}
+        image: "{{ .Values.image.repository40 }}:{{ .Values.image.tag }}"
+        {{end}}
+        ports:
+        - containerPort: 8091
+        - containerPort: 9042
+        - containerPort: 8081
```

Comment:
Some notes here:

* this is missing the public grpc port exposed on the `8090`, it must be exposed, mapped to a service and visible to the outside via the ingress
* the `8091` is the bridge port, this must be exposed, but must not be visible to the outside!

Reply:
Done. As far bridge port visibility, services are configured as ClusterIP so won't be visible outside the cluster.